hudi-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadata.java

```diff
@@ -538,6 +538,10 @@ public Map<String, String> stats() {
     return metrics.map(m -> m.getStats(true, metadataMetaClient, this)).orElse(new HashMap<>());
   }
 
+  public long getMetadataPartitionFileGroupCount(MetadataPartitionType partitionType) {
```

Comment:
why is this change required?

Reply:
i'm gonna move it to test.. it's just for testing

Reply:
Reused an existing method `getNumFileGroupsForPartition`.